tests/test_architectures.py

```diff
@@ -76,7 +76,9 @@ def test_load_timm_model(model_name: str):
     assert (np.max(np.abs(tf_res - pt_res))) / (np.max(np.abs(pt_res)) + 1e-6) < 1e-3
 
 
-@pytest.mark.parametrize("model_name", list_models(module="convnext", exclude_filters=EXCLUDE_FILTERS))
+@pytest.mark.parametrize(
+    "model_name", list_models(module="convnext", exclude_filters=EXCLUDE_FILTERS)
```

Comment:
Could you please change this line to
```
@pytest.mark.parametrize("model_name", list_models(exclude_filters=EXCLUDE_FILTERS))
```
I forgot that I added this, when I was adding the ConvNeXt models....